README.md

```diff
@@ -150,6 +150,7 @@ const russian = voices.russian()
 const german = voices.german()
 const polish = voices.polish()
 const turkish = voices.turkish()
+const canadian = voices.canadian()
```

Comment:
I think this variable name should represent the real language because we can get confused with Canadian and English. In other words, `canadianFrench`, fits better.